drivers/usb/device/Kconfig

```diff
@@ -39,6 +39,39 @@ config USB_DC_STM32
 	help
 	  STM32 family USB device controller Driver.
 
+config USB_DC_STM32_IRQ_PRI
```

Comment:
Please remove this.

Reply:
My mistake. I'll fix this and update it.